platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -307,13 +318,13 @@ public synchronized boolean stopApplication(String appName) {
   @Override
   public void addApplications(List<Map<String, Object>> applicationURLList) {
     for (Map<String, Object> curURL : applicationURLList) {
+      String value = "value";
       try {
-        appService.addApplication(new URI((String) curURL.get("value")));
+        appService.addApplication(new URI((String) curURL.get(value)));
       } catch (URISyntaxException use) {
-        LOGGER.warn("Could not add application with url {}, not a valid URL.", curURL.get("value"));
+        LOGGER.warn("Could not add application with url {}, not a valid URL.", curURL.get(value));
       } catch (ApplicationServiceException ase) {
-        LOGGER.warn(
-            "Could not add application with url {} due to error.", curURL.get("value"), ase);
+        LOGGER.warn("Could not add application with url {} due to error.", curURL.get(value), ase);
```

Comment:
✏️  The exception will not be logged since we are passing parameters in.
❓ Should we not log the exception stack trace at debug level?